src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2799,8 +2841,11 @@ public function attendee_decreases_inventory( array $attendee ) {
 		) {
 			$purchase_time = Tribe__Utils__Array::get( $attendee, 'purchase_time', false );
 
-			if ( false !== $order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $attendee_id ) ) {
-				/** @var \Tribe__Tickets__Commerce__PayPal__Order $order */
+			$order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $attendee_id, [
+				// Get no meta fields.
+			] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: d754d12682d6e4dde92f70ac85ac1296</sub>